prow/cmd/deck/main.go

```diff
@@ -589,6 +591,7 @@ func prodOnlyMain(cfg config.Getter, pluginAgent *plugins.ConfigAgent, authCfgGe
 
 	// Handles link to github
 	mux.HandleFunc("/github-link", HandleGitHubLink(o.github.Host, secure))
+	mux.HandleFunc("generic-provider-link", HandleGenericProviderLink(o.github.Host, secure))
```

Comment:
Make consistent with `/github-link` above?
```suggestion
	mux.HandleFunc("/generic-provider-link", HandleGenericProviderLink(o.github.Host, secure))
```

Reply:
done